### DISCUSSION TITLE
Region grows to cover the whole image and triggers stationary objects...

### DISCUSSION THREAD

**Discussion by Minglarn** [blakeblackshear/frigate#2626] [Ask A Question]
How does "REGION" really work?
The region did grow massively enough to make my cars to trigger as new objects, and therefor a new notification...
As seen in the picture below a person is walking in the "street_area" and triggers both the "yellow" (parking_entrace) and "blue" (parking_area) .... 

In this case the object did not start to get "stationary", but in most events they do and I've a hell of notifications for my stationary cars... 

Why is this happening? This behavior triggers the event to bee sent ... 
![image](https://user-images.githubusercontent.com/58695742/149001740-4a0be2f6-91d2-4edf-993f-82687acfcc70.png)
This is another example... See below.. 
![image](https://user-images.githubusercontent.com/58695742/149007904-4418ade1-5edf-4ead-a282-745994aabd19.png)

This is the MQTT event... What can I do to improve this?
```
{
	"before": {
		"id": "1641929042.781686-m4xx7b",
		"camera": "frigate_parkering",
		"frame_time": 1641929072.046313,
		"snapshot_time": 1641929071.361396,
		"label": "motorfordon",
		"top_score": 0.78125,
		"false_positive": false,
		"start_time": 1641929042.781686,
		"end_time": null,
		"score": 0.7265625,
		"box": [
			413,
			403,
			1070,
			930
		],
		"area": 346239,
		"region": [
			258,
			284,
			1610,
			1636
		],
		"motionless_count": 2,
		"current_zones": [],
		"entered_zones": [
			"lexus_area"
		],
		"has_clip": false,
		"has_snapshot": true
	},
	"after": {
		"id": "1641929042.781686-m4xx7b",
		"camera": "frigate_parkering",
		"frame_time": 1641929072.046313,
		"snapshot_time": 1641929071.361396,
		"label": "motorfordon",
		"top_score": 0.78125,
		"false_positive": false,
		"start_time": 1641929042.781686,
		"end_time": 1641929077.188957,
		"score": 0.7265625,
		"box": [
			413,
			403,
			1070,
			930
		],
		"area": 346239,
		"region": [
			258,
			284,
			1610,
			1636
		],
		"motionless_count": 2,
		"current_zones": [],
		"entered_zones": [
			"lexus_area"
		],
		"has_clip": false,
		"has_snapshot": true
	},
	"type": "end"
}
```

**Comment by Minglarn** [comment]
This is a mp4 about the constant triggering by something ... There is no movement as the motion boxes are not showing up, and yes, they are marked as "show" 
![image](https://user-images.githubusercontent.com/58695742/149189763-87b945ff-52e0-493a-be8f-4494b00306da.png)

But what is making the stationary objects keep triggering as new objects?

https://user-images.githubusercontent.com/58695742/149189436-56a8ce1e-1208-492e-a5e7-5d0553650c76.mp4

**Reply by alexelite** [reply]
I noticed the same thing on the front camera where the cars are parked. No movement but persistent detection frames on that camera.
Debugging showed something similar to yours.
I removed cars from objects being detected and it stopped. Maybe it had something to do with "in progress" events. I am using 0.10.0-DB1255A

**Reply by blakeblackshear** [reply]
Don't forget that presence in a zone is determined by the bottom center of the bounding box. It doesn't matter how much of the object overlaps with the zone. The car on the top left is constantly bouncing in and out of the purple zone because the bottom center of its bounding box is sometimes inside of the zone and sometimes outside of the zone. Also, it does look like zones are only triggered when the object is active which is a bug.

I still need to refine the way "movement" is determined. Right now, its based on how much the bounding boxes moves, not whether or not there is motion overlapping.

It also looks like it sees a 3rd car on the left side of the frame.
